Taskfile.yml

```diff
@@ -290,7 +291,7 @@ tasks:
   controller:test-integration-kind-ci:
     desc: Run live integration tests in kind and deletes the kind cluster afterwards.
     dir: "{{.CONTROLLER_ROOT}}"
-    deps: [controller:kind-create-with-podidentity]
+    deps: [controller:kind-create-helm-podidentity]
```

Comment:
Minor: it might not hurt to take a page from @babbageclunk's book and stick a `-v` on the `go test -timeout 15m` line down below, to make it: `go test -timeout 15m -run -v ...`

Reply:
No need to do that in this PR though necessarily. Feel free to merge this!